drivers/include/net/netdev.h

```diff
@@ -249,6 +249,7 @@ typedef enum {
 struct netdev_radio_rx_info {
     int16_t rssi;       /**< RSSI of a received packet in dBm */
     uint8_t lqi;        /**< LQI of a received packet */
+    uint8_t crc_valid;  /**< CRC was valid (when != 0) */
```

Comment:
Now this might be in the "premature optimization" area. Is there a use case for changing this to `uint8_t flags;` and use it as a number of boolean flags for received frames, something like
```
NETDEV_RADIO_RX_CRC_VALID   (0x01) /**< Set if the CRC was valid */
NETDEV_RADIO_RX_DATA_PEND  (0x02) /**< Set if the received frame has the data pending bit set */
```

I don't want to stall this PR more than necessary with this idea. if we *might* need this in the future, having a `bool netdev_radio_crc_valid(netdev_radio_rx_info_t *info)` getter for the crc status would prevent a migration to flags being a significant API change.

Reply:
Sounds valid to me, @miri64 what's your opinion?

Reply:
Agreed, however also see https://github.com/RIOT-OS/RIOT/pull/11023 for pending data handling ;) (since that can be read from the MAC header at least in IEEE 802.15.4's case, I'm not sure it needs to be in the info struct ;-)